onnx/defs/quantization/old.cc

```diff
@@ -110,7 +110,10 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Output(0, "y", "N-D full precision output tensor. It has same shape as input 'x'.", "T2")
         .Attr(
             "axis",
-            "(Optional) The axis of the dequantizing dimension of the input tensor. Ignored for per-tensor quantization. Negative value means counting dimensions from the back. Accepted range is [-r, r-1] where r = rank(input).",
+            "(Optional) The axis of the dequantizing dimension of the input tensor. Used only for per-axis and blocked "
```

Comment:
Blocked quantization is not supported in the old opset: [Blocked quantization](https://github.com/onnx/onnx/blob/c459890aa266d74ca612f334d0f3e869dcdbb597/onnx/version_converter/adapters/q_dq_21_20.h#L63C30-L63C50)

Reply:
Thanks so much!! Will update this asap

Reply:
Was this updated?

Reply:
Thanks for double checking this! Just removed “blocked quantisation” from old.cc